source/Assets/Plugins/Didomi/Scripts/Android/AndroidObjectMapper.cs

```diff
@@ -430,7 +430,7 @@ public static AndroidJavaObject ConvertToJavaUserAuthParams(UserAuthParams param
             {
                 UserAuthWithEncryptionParams encryptionParameters = (UserAuthWithEncryptionParams)parameters;
                 return new AndroidJavaObject(
-                    "io.didomi.sdk.user.UserAuthWithEncryptionParams",
+                    "io.didomi.sdk.user.model.UserAuthWithEncryptionParams",
```

Comment:
Arg... Didn't though of this issue when I changed the package

Reply:
There was the same issue in Flutter and react-native, too bad we didn't catch it during the code review. We didn't hear from it from customers team so i guess clients figured out to do the change.